### DISCUSSION TITLE
Nested Array of Object not working.

### DISCUSSION THREAD

**Discussion by danyhiol** [vazco/uniforms#1115] [Q&A] [answered]
I am trying to render something like [this](https://uniforms.tools/playground/#?N4IgDgTgpgzlAuIBcIDqUA2BjA9gWygEIQAacCHMGZUAQwFd4cZaA3KZAM1ozjIaYt2AEUy0AnsgCMABhlkAJgEsWAIwxQFXHnxAZaqzMngR6UMmH1YoACxwYFUCNt7mQ0WgoDyAOwySkblcyGCwbKDxaZBAfKAB3AAIAZSU8SygksIjaACYAIQglBQBzKAAKAB0fBITYxJS0jUzwyLLgKpqahVp4KCQE4AT4cTA-hOEe8wTHbnoMeAA1HjN-uvHJsoBKBIBfEg7Ozzn4fryceyhaH33qzpglAC8x9tvOoZGxpJMlH2Kbt66UFm8yWGBWCQA5HgIf8ATwMDg4ppQWYYP0ANoQgAeMBhkNxJEh0MJEIweOxZIAurCansDjUID0fsV-i8Ae9Rv0AHL0PCGCA0w4YBFIhQo2AYqSEnKEgDMhIALISAKzU-mdeg-JScHAQPBogYJLJYADWqhwWIlQ1MUF2grprwSnEKUB8CgNg2GnISAEEIIzxIS8D8AMI4TUnBJSO3qiHOpSu90AOgAJBD-l5VAArKBYeCCuMut0wVNJny0Ajpw1ez7fX5Bn79WUxx2FhPF0u0UpVz0fbm8_kNnz9eQJSJY_pSZUyFtvNuJksppO9GDwHscsZ-gOzzrzjtLlfwVPrmv9L6FX47nabKqbUggGB2OIAST8PygAFF_bqXLpaDBxB8LBfCSehVGDRBAh0Nx_0A4CfFBIpJl_KA9hAeAWg4FArngLQdiAA) using `SimpleSchema`;

```
friends: { type: Array, minCount: 1 },
'friends.$': Object,
'friends.$.name': { type: String, min: 3 },
'friends.$.age': { type: Number, min: 0, max: 150 },
'friends.$.test': { type: Array },
'friends.$.test.$': { type: String },
```
But I am getting:` TypeError: Cannot read properties of undefined (reading 'length')` coming from : `const limitNotReached = !disabled && !(parent.maxCount <= parent.value.length);`. Maybe an existance check should be apply on parent first? -> 
`const limitNotReached = !disabled && parent && !(parent.maxCount <= parent.value.length);`

**Comment by radekmie** [comment, marked as answer]
Hi @danyhiol. That's a duplicate of #1112 and it's being worked on.
